--- qt-everywhere-src-6.0.1/qtbase/src/network/configure.cmake	2021-01-26 18:29:22.000000000 +0000
+++ qt-everywhere-src-6.0.1.mod/qtbase/src/network/configure.cmake	2021-02-21 01:28:52.612701209 +0000
@@ -437,5 +437,5 @@
 qt_configure_add_report_entry(
     TYPE NOTE
     MESSAGE "When linking against OpenSSL, you can override the default library names through OPENSSL_LIBS. For example: OPENSSL_LIBS='-L/opt/ssl/lib -lssl -lcrypto' ./configure -openssl-linked"
-    CONDITION NOT ANDROID AND QT_FEATURE_openssl_linked AND ( NOT TEST_openssl.source EQUAL 0 ) AND INPUT_openssl.prefix STREQUAL '' AND INPUT_openssl.libs STREQUAL '' AND INPUT_openssl.libs.debug STREQUAL '' OR FIXME
+    CONDITION (TERMUX OR NOT ANDROID) AND QT_FEATURE_openssl_linked AND ( NOT TEST_openssl.source EQUAL 0 ) AND INPUT_openssl.prefix STREQUAL '' AND INPUT_openssl.libs STREQUAL '' AND INPUT_openssl.libs.debug STREQUAL '' OR FIXME
 )
